latitudesh/terraform-provider-latitudesh
# Feat: Add billing attribute to server creation

#### What does this PR do?
Add the billing option to server resource

#### Description of Task to be completed?
- [x] Add billing attribute to server resource
- [x] Add provisioning type attribute to project resource
- [x] Update Go SDK to version 0.4.3

#### How should this be manually tested?
If https://github.com/latitudesh/latitudesh-go/pull/49 isn't merged yet, you should get the updated SDK version by running:
```
go get github.com/latitudesh/latitudesh-go@PD-4037-SDK-Provider-Add-billing-attribute-to-server-creation
```
Them you can proceed with testing:

You should run the testing suit to guarantee that the changes aren't breaking any existing functionality.
```
export LATITUDESH_AUTH_TOKEN=<YOUR-API-TOKEN> 
```
```
TF_ACC=1 go test ./... -run "TestAccProject" -v
```
```
TF_ACC=1 go test ./... -run "TestAccServer" -v
```

also, you can follow [Notion - Developing the Terraform Provider Locally](https://www.notion.so/Developing-the-Terraform-Provider-Locally-6682fe3df39d4003b519301e630f51ef?pvs=27&qid=1%3A8c80df71-563c-4c6a-905d-be4a225ec805%3A0) to run the provider locally.

#### Any background context you want to provide?
This PR depends on https://github.com/latitudesh/latitudesh-go/pull/49.
